web/src/js/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import * as modalAction from '../../ducks/ui/modal'
+import ModalList from './ModalList'
+
+class PureModal extends Component {
+
+    constructor(props, context) {
+        super(props, context)
+    }
+
+    render() {
+        const { activeModal, hideModal } = this.props
+        const ActiveModal = _.find(ModalList, m => m.name === activeModal )
```

Comment:
_.find returns undefined if the element is not found 
https://lodash.com/docs/#find
you can rewrite this to 

> const ActiveModal = _.find(ModalList, m => m.name === activeModal ) || Fallback case (<div/>) <
then you can just return ActiveModal

>